src/test/java/saps/common/core/storage/swift/SwiftAPIClientTest.java

```diff
@@ -16,9 +16,10 @@
 
   private Properties loadConfigFile(String path) throws IOException {
     Properties properties = new Properties();
-    FileInputStream input = new FileInputStream(path);
-    properties.load(input);
-    return properties;
+    try (FileInputStream input = new FileInputStream(path)) {
```

Comment:
same here. I didn't get the point of a catch-less try

Reply:
Same: the purpose is to close ensure the FileInputStream will be closed outside this scope.